src/components/Content/Content.tsx

```diff
@@ -0,0 +1,12 @@
+import { Route, Routes } from "react-router-dom";
+import SignInPage from "../../pages/SignInPage/SignInPage";
+import DiaryHome from "../../pages/DiaryHomePage/DiaryHome";
+
+export default function Content() {
+  return (
+    <Routes>
+      <Route path="/" element={<SignInPage />} />
```

Comment:
store routes in a constant file and use that, don't hardcode routes everywhere. and also rename this file as RootRouter or something.